mypy/checker.py

```diff
@@ -7994,11 +7994,13 @@ def conditional_types(
 ) -> tuple[Type | None, Type | None]:
     """Takes in the current type and a proposed type of an expression.
 
-    Returns a 2-tuple: The first element is the proposed type, if the expression
-    can be the proposed type. The second element is the type it would hold
-    if it was not the proposed type, if any. UninhabitedType means unreachable.
-    None means no new information can be inferred. If default is set it is returned
-    instead."""
+    Returns a 2-tuple:
+        The first element is the proposed type, if the expression can be the proposed type.
+        The second element is the type it would hold if it was not the proposed type, if any.
+        UninhabitedType means unreachable.
+        None means no new information can be inferred.
+        If default is set it is returned instead.
```

Comment:
Maybe this line shouldn't be separated from the previous one? Now it's a bit ambiguous whether default is returned instead of `None` only or instead of both `None` and `Uninhabited`